### PR TITLE
test: fix git commit hook tests on python3.6

### PR DESCRIPTION
**Pull request by Panaetius**
Fixes locale problem with pytest, gitpython, click and python3.6
Fixes GitHookException stdout in case where messages are too long.

### CODE REVIEW

File: renku/data/pre-commit.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ IFS=$'\n' read -r -d '' -a MODIFIED_FILES \
   <<< "$(git diff --name-only --cached --diff-filter=M)"
 IFS=$'\n' read -r -d '' -a ADDED_FILES \
   <<< "$(git diff --name-only --cached --diff-filter=A)"
-
+echo "a"
```
Comment (mohammad-sdsc) [inline renku/data/pre-commit.sh:30]: Debugging left-over?

File: renku/data/pre-commit.sh [hunk 2]
```diff
@@ -66,7 +66,7 @@ if [ ${#MODIFIED_FILES[@]} -ne 0 ] ; then
     echo 'To commit anyway, use "git commit --no-verify".'
     exit 1
   fi
-  IMMUTABLE_TEMPLATE_FILES=$(renku migrate check-immutable-template-files "${MODIFIED_FILES[@]}")
+  IMMUTABLE_TEMPLATE_FILES=$(renku check-immutable-template-files "${MODIFIED_FILES[@]}")
```
Comment (mohammad-sdsc) [inline renku/data/pre-commit.sh:69]: 😱

File: renku/data/pre-commit.sh (hunk 1 above)
Comment (mohammad-sdsc) [reply renku/data/pre-commit.sh:30]: In some other places as well.

**Review by mohammad-sdsc — COMMENTED** [review]
Made some comments! I was wondering if we should use Python 3.6 as default in PR CI instead of 3.7.

File: renku/data/pre-commit.sh (hunk 2 above)
Comment (Panaetius) [reply renku/data/pre-commit.sh:69]: right... :fearful:

**Review by mohammad-sdsc — APPROVED** [review]
LGTM, thanks!

**Pull request merged**